src/mon/OSDMonitor.cc

```diff
@@ -823,7 +823,7 @@ class OSDUtilizationDumper : public CrushTreeDumper::Dumper<F> {
 
     for (int k = osdmap->crush->get_bucket_size(id) - 1; k >= 0; k--) {
       int item = osdmap->crush->get_bucket_item(id, k);
-      int64_t kb_i = 0, kb_used_i = 0, kb_avail_i;
+      int64_t kb_i = 0, kb_used_i = 0, kb_avail_i = 0;
```

Comment:
i just realized this is not a bug. as we already initialize them. could you remove the "correctly" in the commit message if you are with me.